packages/bot/src/services/api/contracts-for.js

```diff
@@ -11,13 +11,28 @@ export default class ContractsFor {
         // e.g. if market omitted, rule will apply to all markets.
         this.disabled_options          = [
             {
-                market    : 'forex',
                 submarket : 'smart_fx',
                 trade_type: 'higherlower',
             },
+            {
+                submarket : 'smart_fx',
+                trade_type: 'callputequal',
+            },
             {
                 submarket : 'minor_pairs',
-                trade_type: 'higherlower',
```

Comment:
`higherlower` is also not available for `minor_pairs`, this disabled option needs to be re-added.